vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt

```diff
@@ -271,6 +272,7 @@ class BugReporter @Inject constructor(
                     deviceId = session.sessionParams.deviceId ?: "undefined"
                     olmVersion = session.cryptoService().getCryptoVersion(context, true)
                     bugReportURL = session.sessionParams.homeServerUrl.removeSuffix("/") + BUG_REPORT_URL_SUFFIX
+                    email = session.profileService().getThreePids().filterIsInstance<ThreePid.Email>().firstOrNull()?.email ?: "undefined" // TCHAP Add Email
```

Comment:
I don't know the comment convention to add.
Also It's a suggestion from reading the code, I haven't compiled and tested it yet.